src/components/organize/TagManager/TagManagerController.tsx

```diff
@@ -29,7 +29,7 @@ export const TagManagerController: React.FunctionComponent<
   availableTags,
   disableEditTags,
   disabledTags,
-  groupTags = false,
+  groupTags = true,
```

Comment:
Is it necessary to change both this, and the state in `TagManagerSection`? That means we're changing the default "on two levels", and I'm not sure that's what we want.

Can you explain, @ziggabyte?

Reply:
i don't think it's neccessary, but I thought go big or go home and changed everywhere lol